tools/WinObjC.Tools/WinObjC.Tools.nuproj

```diff
@@ -19,13 +19,13 @@
     <Authors>Microsoft</Authors>
     <Owners>Microsoft</Owners>
     <Summary>Command Line tools and Visual Studio Extensions for the WinObjC Project.</Summary>
-    <Description>Command Line tools and Visual Studio Extensions for the WinObjC Project. This package includes vsimporter, the tool to convert an existing Xcode Cbjective-C project to a Visual Studio solution, as well as the Visual Studio Extensions (requires Visual Studio 2017 or later) for Objective C syntax highlighting and Nuget package creation. See https://github.com/Microsoft/WinObjC for more details.</Description>
+    <Description>Command Line tools and Visual Studio Extensions for the WinObjC Project. This package includes vsimporter ( the tool to convert an existing Xcode Cbjective-C project to a Visual Studio solution), winmd2objc (the tool to generate ObjC projections from Windows-MetaData files (winmd)), as well as the Visual Studio Extensions (requires Visual Studio 2017 or later) for Objective C syntax highlighting and Nuget package creation. See https://github.com/Microsoft/WinObjC for more details.</Description>
```

Comment:
nit: `Windows-MetaData` has an unnecessary hyphen _and_ `metadata` should never be camel case. `Windows Metadata`